client/templates/dashboard/dashboard.js

```diff
@@ -1,3 +1,7 @@
+Template.dashboardShopifyProducts.onCreated(function () {
+  this.subscribe('ShopifyProducts/CurrentImport');
+});
+
```

Comment:
General question - is it better to subscribe onCreated? Or in the route to waitOn the Subscription?